### PR TITLE
[Components with more than 1 service] LPS-175891, LPS-176008, LPS-176076

### PR DESCRIPTION
**Pull request by daiazius**
Related issues: https://issues.liferay.com/browse/LPS-175891, https://issues.liferay.com/browse/LPS-176008, https://issues.liferay.com/browse/LPS-176076

### CODE REVIEW

**Comment by daiazius** [comment]
ci:test:startup

**Comment by daiazius** [comment]
ci:test:commerce

**Comment by daiazius** [comment]
ci:test:object

**Comment by rafaprax** [comment]
@daiazius  Let's apply the same pattern here https://github.com/liferay-core-infra/liferay-portal/pull/3866

**Comment by rafaprax** [comment]
ci:stop

**Comment by daiazius** [comment]
ci:test:sf

**Comment by daiazius** [comment]
Pushed with no changes in the files, I've only changed the commit pattern

**Comment by catarineaguiar** [comment]
Will retrigger test suites because most were aborted

**Comment by catarineaguiar** [comment]
ci:test:commerce

**Comment by catarineaguiar** [comment]
ci:test:object

File: modules/dxp/apps/commerce/headless/headless-commerce-machine-learning-impl/src/main/java/com/liferay/headless/commerce/machine/learning/internal/resource/v1_0/AccountCategoryForecastResourceImpl.java [hunk 1]
```diff
@@ -151,9 +151,12 @@ public void create(
 					historyLength, forecastLength));
 	}
 
-	@Reference
-	private AccountCategoryForecastDTOConverter
-		_accountCategoryForecastDTOConverter;
+	@Reference(
```
Comment (rafaprax) [inline modules/dxp/apps/commerce/headless/headless-commerce-machine-learning-impl/src/main/java/com/liferay/headless/commerce/machine/learning/internal/resource/v1_0/AccountCategoryForecastResourceImpl.java:154]: @daiazius, this change should not be in this commit.

**Review by rafaprax — CHANGES_REQUESTED** [review]

**Review by rafaprax — APPROVED** [review]

**Comment by catarineaguiar** [comment]
no regressions found.

**Pull request closed without merging**